---
Rokos_Task:
  # None = Full Task
  # Full task almak icin diger taskleri yoruma alÄ±p, None'i ac

  Right_Rokos: None
    #- 35272
    #- 35277
    #- 35279
    #- 35291
    #- 35310
    #- 35320
    #- 35330
    #- 35340
    #- 35349
    #- 35360
    #- 35374
    #- 35380
    #- 35390
    #- 35414
    #- 35432

  Left_Rokos: None
    #- 35702
    #- 35705
    #- 35700
    #- 35710
    #- 35730
    #- 35749
    #- 35760
    #- 35770
    #- 35780
    #- 35790
    #- 35818
    #- 35825
    #- 35830
    #- 35840
    #- 35875
